pkg/geo/geo.go

```diff
@@ -11,11 +11,127 @@
 package geo
 
 import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/cockroachdb/cockroach/pkg/geo/geopb"
+	"github.com/cockroachdb/errors"
+	"github.com/golang/geo/s2"
+	"github.com/twpayne/go-geom"
+
 	// Force these into vendor until they're used.
-	_ "github.com/golang/geo/s2"
-	_ "github.com/otan-cockroach/gogeos/geos"
-	_ "github.com/twpayne/go-geom"
-	_ "github.com/twpayne/go-geom/encoding/ewkb"
-	_ "github.com/twpayne/go-geom/encoding/ewkbhex"
+	"github.com/otan-cockroach/gogeos/geos"
+	"github.com/twpayne/go-geom/encoding/ewkb"
+	"github.com/twpayne/go-geom/encoding/ewkbhex"
 	_ "github.com/twpayne/go-geom/encoding/wkbhex"
 )
+
+var featureEncodingEndianness = binary.LittleEndian
+
+// Geometry is planar spatial object.
+type Geometry struct {
+	// TODO(dan): This should be a geos.Geometry.
+	Shape geom.T
+	SRID  geopb.SRID
+}
+
+// Geography is a spherical spatial object.
+type Geography struct {
+	Shape s2.Region
```

Comment:
i think this should be []s2.Region, in case of `GeometryCollection`, `MultiPoint`, `MultiLineString` and `MultiPolygon`

Reply:
good point, i'll do this. geom.T is fine, right?

Reply:
yep.

Reply:
done